app/assets/javascripts/rules/story.js

```diff
@@ -0,0 +1,26 @@
+const iconRules = {
```

Comment:
We should have tests for all the rules in this file 😄

Reply:
What do you think to implement jest to rest react components?

Reply:
I don't like the idea of mixing two testing frameworks during a transition of the approach like we're doing right now, maybe after the whole transition we can migrate completely from Jasmine to Jest but there's not a big gain right now.

Reply:
OK